core/variant/variant.h

```diff
@@ -532,6 +532,7 @@ class Variant {
 		OP_AND,
 		OP_OR,
 		OP_XOR,
+		OP_NAND,
```

Comment:
I think `NAND` is a rarer operation than even `XOR`. For `NAND`, it is better to open a separate proposal first. If it gets support, `NAND` can be added in a separate PR.